pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/player/compose/TvPlayerView.kt

```diff
@@ -60,46 +63,55 @@ fun TvPlayerView(
     ) {
-        PlayerSurface(
-            player = player,
-            modifier = Modifier
-                .fillMaxSize()
-                .handleDPadKeyEvents(onEnter = {
-                    visibilityState.show()
-                })
-                .focusable(true)
-        )
-        AnimatedVisibility(
-            visible = visibilityState.isVisible,
-            enter = expandVertically { it },
-            exit = shrinkVertically { it }
-        ) {
-            Box(
+        val error by player.playerErrorAsState()
+        if (error != null) {
+            CompositionLocalProvider(
+                LocalContentColor provides MaterialTheme.colorScheme.onSurface
+            ) {
+                PlayerError(modifier = Modifier.fillMaxSize(), playerError = error!!, onRetry = player::prepare)
+            }
+        } else {
+            PlayerSurface(
+                player = player,
                 modifier = Modifier
                     .fillMaxSize()
-                    .maintainVisibleOnFocus(delayedVisibilityState = visibilityState),
-                contentAlignment = Alignment.Center
+                    .handleDPadKeyEvents(onEnter = {
+                        visibilityState.show()
+                    })
+                    .focusable(true)
+            )
+            AnimatedVisibility(
+                visible = visibilityState.isVisible,
+                enter = expandVertically { it },
+                exit = shrinkVertically { it }
             ) {
-                TvPlaybackRow(
-                    player = player,
-                    state = visibilityState
-                )
-
-                IconButton(
-                    onClick = { drawerState.setValue(DrawerValue.Open) },
+                Box(
                     modifier = Modifier
-                        .padding(MaterialTheme.paddings.baseline)
-                        .align(Alignment.BottomEnd)
+                        .fillMaxSize()
+                        .maintainVisibleOnFocus(delayedVisibilityState = visibilityState),
+                    contentAlignment = Alignment.Center
                 ) {
-                    Icon(
-                        imageVector = Icons.Default.Settings,
-                        contentDescription = stringResource(R.string.settings)
+                    TvPlaybackRow(
+                        player = player,
+                        state = visibilityState
                     )
+
+                    IconButton(
+                        onClick = { drawerState.setValue(DrawerValue.Open) },
+                        modifier = Modifier
+                            .padding(MaterialTheme.paddings.baseline)
+                            .align(Alignment.BottomEnd)
+                    ) {
+                        Icon(
+                            imageVector = Icons.Default.Settings,
+                            contentDescription = stringResource(ch.srgssr.pillarbox.demo.tv.R.string.settings)
```

Comment:
```suggestion
                            contentDescription = stringResource(R.string.settings)
```

With an import on `import ch.srgssr.pillarbox.demo.tv.R`.